microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+---
```

Comment:
clean up build warnings
https://github.com/MicrosoftDocs/edge-developer/pull/2137#issuecomment-1230734294

Reply:
Yes, i am aware of those thanks.  We wanted to get this to the PMs beforefixing those.  And you told me to do those last, but I am on it.